pkg/manifest/ostree_deployment.go

```diff
@@ -481,3 +494,34 @@ func (p *OSTreeDeployment) getInline() []string {
 
 	return inlineData
 }
+
+// Creates systemd unit stage by ingesting the servicename and mount-points
+func createMountpointService(serviceName string, mountpoints []string) *osbuild.SystemdUnitCreateStageOptions {
+	var conditionPathExists []string
+	for _, mountpoint := range mountpoints {
+		conditionPathExists = append(conditionPathExists, "|!"+mountpoint)
+	}
+	unit := osbuild.Unit{
+		Description:         "Generated by osbuild/images - Create mountpoints if does not exist",
```

Comment:
Small syntax fix and changes to `osbuild` since `osbuild/images` doesn't really mean much in the end (it's osbuild that's doing the building and writing of files)
```suggestion
		Description:         "Generated by osbuild - Create mountpoints if they do not exist",
```